### ISSUE TITLE
Hosting the ViteNET.MVC example as a subsite under IIS

### ISSUE THREAD

**Issue by jackkjhsu** [Eptagone/Vite.AspNetCore#13]
Hi,

I'm trying to host the ViteNET.MVC example under an IIS site with the path "/vite", i.e. http://localhost/vite. This subsite runs as "Production" as no environmentVariable is set on the web.config file.
I could not get the css and js tags to generate by the Vite middleware. However if I deploy this vite app as a top level IIS site, i.e. http://localhost:888, then everything works as expected.

I am getting the following messages from the sdtout logs:
```
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: C:\TestProjects\Vite.AspNetCore\examples\Deployment\vite\
warn: Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware[3]
      Failed to determine the https port for redirect.
warn: Vite.AspNetCore.Services.ViteManifest[0]
      The chunk 'vite/main.ts' was not found.
fail: Vite.AspNetCore.TagHelpers.ViteTagHelper[0]
      "vite/main.ts" was not found in Vite manifest file (check /Views/Home/Index.cshtml)
warn: Vite.AspNetCore.Services.ViteManifest[0]
      The chunk 'vite/main.ts' was not found.
fail: Vite.AspNetCore.TagHelpers.ViteTagHelper[0]
      "vite/main.ts" was not found in Vite manifest file (check /Views/Home/Index.cshtml)
```
Is there anything I can do to specify a base path so that the middleware can find the asset files properly? Ideally, the middleware should be able to "autodetect" the base path according to the incoming requests.

Thanks for your help.

**Comment by jackkjhsu** [comment]
More Info:
If I add an https binding to the parent site localhost, making the URL https://localhost:444/vite, the errors above go away and I get the css and js tags generated as follows but the relative paths are incorrect and should start with "/vite/".
![image](https://user-images.githubusercontent.com/5721138/235031507-5783502f-2307-46d9-b24a-8d5144b16815.png)

Thanks for your help again.

**Comment by jackkjhsu** [comment]
I think I know where the issue lies, within the `ViteTagHelper.Process` method. I'll do more poking around and some tests. If it turns out that a change is really needed, and with your permission, I'll create a PR to address the base path issue for all use cases.

Thank you all.

**Comment by Eptagone** [comment]
Hello @jackkjhsu. Thanks for your report.
You're right. The slash is missing. And that's why it doesn't work without https.
I'll fix it right now.

**Comment by Eptagone** [comment]
Done. Fixed with c7804d7

**Comment by jackkjhsu** [comment]
Thank you very much Quetzal.
